Gemfile

```
@@ -3,27 +3,17 @@ source 'https://rubygems.org'
 gem 'kafo', '~> 4.1'
 gem 'librarian-puppet'
 gem 'puppet', ENV.key?('PUPPET_VERSION') ? "~> #{ENV['PUPPET_VERSION']}" : '~> 6.0'
-if RUBY_VERSION < '2.1'
-  gem 'puppet-strings', '< 2.0.0'
-else
-  gem 'puppet-strings'
-end
+gem 'facter', '~> 4.0'
```

Technically it should work with Facter 3 as well and in some installs you can use that. It just doesn't work with Facter 2 anymore.
```suggestion
gem 'facter', '>= 3.0'
```